python/eos/analysis.py

```diff
@@ -702,7 +702,7 @@ def _prior_transform(self, u):
         return self._u_to_par(u)
 
 
-    def sample_nested(self, bound='multi', nlive=250, dlogz=1.0, maxiter=None, seed=10, print_progress=True, sample='auto'):
+    def sample_nested(self, bound='multi', nlive=250, dlogz=1.0, maxiter=None, miniter=0, print_progress=True, print_function=None, seed=10, sample='auto'):
```

Comment:
Can you add a docstring for `print_function`?

Reply:
Done on the ``Analysis`` side.